cmd/common_test.go

```diff
@@ -110,3 +113,135 @@ func TestNewRunFilter(t *testing.T) {
 	})
 
 }
+
```

Comment:
I like the fact the testing using file data. I think it'll be better to have the contents to be in its own files, like `master_only.yaml`, `node_only.yaml`, `master_node.yaml`. The benefit I see of having external files is that that it'll be easier to check of file format and modifications as these files change later one.

Reply:
Addressed.